src/deroff.rs

```diff
@@ -1,10 +1,40 @@
 /// A translation of https://github.com/fish-shell/fish-shell/blob/e7bfd1d71ca54df726a4f1ea14bd6b0957b75752/share/tools/deroff.py
 // """ Deroff.py, ported to Python from the venerable deroff.c """
+extern crate regex;
+use regex::Regex;
 
 // class Deroffer:
-struct Deroffer {}
+struct Deroffer {
+    g_re_word: &'static Regex,
+    g_re_number: &'static Regex,
+    g_re_esc_char: &'static Regex,
```

Comment:
Doesn't look like `g_re_esc_char` is used anywhere. Can remove?

Reply:
Done in 5e03a24 :-)